pegasus/migrations/138_rename_storage_apps.rb

```diff
@@ -0,0 +1,41 @@
+Sequel.migration do
+  up do
+    pegasus_storage_apps = "#{CDO.pegasus_db_name}__storage_apps".to_sym
+    dashboard_projects = "#{CDO.dashboard_db_name}__projects".to_sym
+
+    unless [:production].include?(rack_env)
+      begin
+        # Rename the table to move it from the Pegasus schema to Dashboard
+        # Also change the name from storage_apps to projects
+        rename_table(pegasus_storage_apps, dashboard_projects)
+
+        # This view exists as a safe-guard during the time between when the table rename is applied
+        # And the code referencing the new name is deployed, during that time queries to the old table name
+        # will hit this view and query from the renamed table
+        create_view(pegasus_storage_apps, DB[dashboard_projects].select_all)
+      rescue Sequel::DatabaseError => e
+        is_table_already_exists_error = e.message.include? "Table 'projects' already exists"
+        is_pegasus_table_empty = DB[pegasus_storage_apps].count == 0
+        # When setting up environments from scratch, the new table will be created from the dashboard schema.rb
+        # and there will be no previously existing storage_apps data, so the old pegasus table can be dropped.
+        # The view will be created for consistency across environments.
+        if is_table_already_exists_error && is_pegasus_table_empty
+          puts "projects has already been created, dropping pegasus storage_apps..."
```

Comment:
It might be safer for us to use `CDO.log` instead of `puts`. We have logic that treats and usage of `puts` in a cron job as an error that should result in a HoneyBadger notification and this migration will be run by our automated build job (`/aws/ci_build`) on non-production managed environments.

Reply:
I [could be wrong](https://github.com/code-dot-org/code-dot-org/blob/da1318a5b93303532c2e28628711a3d4f335689f/lib/cdo/honeybadger.rb#L37) about `puts` (standard out) triggering a HoneyBadger error, but still probably more consistent for us to use `CDO.log` whenever possible.